README.md

```diff
@@ -20,14 +20,24 @@ Supported platforms:
 
 ## Usage
 
-This repository contains C-language examples that link to the native library targets and perform basic rendering and computation.
-To run the C triangle example, install a C compiler + glfw 3, then run these commands at the root of the repo:
+This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. The prerequisites are a C compiler and GLFW 3 for running the examples.
+
+### Running an example
+The examples are located under the [examples](examples) directory. We are using a [Makefile](Makefile) for running the examples.
+
+#### Triangle
+```bash
+make example-triangle
 ```
-rustup toolchain install nightly
-cargo install cbindgen
```

Comment:
why are we removing `cbindgen` installation instructions?

Reply:
My understanding was that `cbindgen` was included in the Cargo.toml, but as I am looking at it now it isn't. I will add some more documentation about having this installed as prerequisite.